flocker_bb/builders/flocker_vagrant.py

```diff
@@ -182,7 +182,10 @@ def buildTutorialBox():
 def run_acceptance_tests(distribution, provider):
     factory = getFlockerFactory()
     factory.addSteps(_flockerTests(
-        kwargs={'trialMode': []},
+        kwargs={
+            'trialMode': [],
+            'sigtermTime': 5*60,
```

Comment:
Worth adding a comment explaining why this is here? E.g. Allow 5 minutes for acceptance test machines to be removed.